src/App.js

```diff
@@ -1,30 +1,44 @@
 import React from 'react';
+import { PostList } from './components/PostList/PostList';
 
 import './App.scss';
 
-import posts from './api/posts';
-import comments from './api/comments';
-import users from './api/users';
+import postsFromServer from './api/posts';
+import commentsFromServer from './api/comments';
+import usersFromServer from './api/users';
+
+const preperedPosts = postsFromServer.map(post => ({
+  ...post,
+  user: usersFromServer.find(user => user.id === post.userId),
+  commentInPost: commentsFromServer.filter(comment => (
```

Comment:
```suggestion
  commentsInPost: commentsFromServer.filter(comment => (
```
there is a few of them

Reply:
Yes, I agree